student.rb

```diff
@@ -1,15 +1,14 @@
 require './person'
+
 class Student < Person
-  def initialize(classroom)
-    super
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
     @classroom = classroom
   end
 
```

Comment:
- [ ] In this class you should implement `belongs-to` relationships between `Classroom` and `Student`, kindly make sure to implement this so the associations are set up correctly

e.g
```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```